testament/categories.nim

```diff
@@ -459,11 +459,12 @@ proc testNimblePackages(r: var TResults; cat: Category; packageFilter: string) =
       r.addResult(packageFileTest, targetC, "", "", reBuildFailed)
 
   except JsonParsingError:
-    echo "[Warning] - Cannot run nimble tests: Invalid package file."
+    errors = 1
```

Comment:
```suggestion
    echo "[Warning] - Cannot run nimble tests: Invalid package file."
```

Reply:
But that's taken from your PR.

Reply:
My PR raise exception at this situation.